.gitignore

```diff
@@ -1,90 +1,39 @@
 # Temporary Build Files
```

Comment:
Could we keep this gitignore files, I see a lot iterms are removed.

Reply:
AFAIK those were auto-generated by old o-sdk. The .gitignore generated by the new o-sdk didn't have these lines.